includes/discovery/processors/terra.inc.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+if ($device["os"] === "terra") {
+    $descr = "Processor";
+
+    $query = array(
+       array("sti410C", ".1.3.6.1.4.1.30631.1.9.1.1.3.0"),
+       array("sti440",  ".1.3.6.1.4.1.30631.1.18.1.326.3.0")
+    );
+
+    $sysDescr = strlen($device["sysDescr"]) == 0 ? snmp_get($device, ".1.3.6.1.2.1.1.1.0", "-Ovq") : $device["sysDescr"];
```

Comment:
$device["sysDescr"] is empty after adding and first discovery (at cron: /opt/librenms/discovery.php -h new). sysDescr is available only after manual discovery or after /opt/librenms/discovery.php -h all. Without that line first graphs are shown only after the discovery.php -h all (after a few hours in the worst case).